### DISCUSSION TITLE
Where can I find the command list?

### DISCUSSION THREAD

**Discussion by erichabg** [Koenkk/zigbee2mqtt#6677] [General]
I have searched the documentation but I did not find a list with the supported commands.
Especially I need the command to get a list of all zigbee devices. I think it is "zigbee2mqtt/bridge/config/devices/get 123456"
This command I have found in the nodered Flow Z2M Admin. But this flow doesn't work anymore (no devices in device list) and now I'm trying to get it work again.
When I send "zigbee2mqtt/bridge/config/devices/get 123456" the result is of type "groups" and therefore the device list remains empty.  
Also a description of the output would be fine.
Regards  Erich

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html
